lib/core/network/router.js

```diff
@@ -138,6 +138,33 @@ class Router {
         this._executeFromHttp(route.verb, request, cb);
       });
     }
+
+    /**
+     * Returns inner metrics from the router
+     * @returns {Object}
+     */
+    global.kuzzle.onAsk(
+      'core:network:router:metrics',
+      () => this.metrics());
+  }
+
+  /**
+   * Returns the metrics of the router
+   * @returns {Object}
+   */
+  metrics () {
+    const connectionsByProtocol = new Map();
+    for (const [, connection] of this.connections) {
```

Comment:
You could iterate over the values only with [Map.values](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/values)